ionoscloud/resource_target_group_test.go

```diff
@@ -50,6 +51,7 @@ func TestAccTargetGroupBasic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceNameTargetGroup, "targets.0.ip", "22.231.2.2"),
 					resource.TestCheckResourceAttr(resourceNameTargetGroup, "targets.0.port", "8080"),
 					resource.TestCheckResourceAttr(resourceNameTargetGroup, "targets.0.weight", "1"),
+					resource.TestCheckResourceAttr(resourceNameTargetGroup, "targets.0.proxy_protocol", "v2"),
```

Comment:
@cbeti-ionos it doesn't have anything to do with the scope of this PR, but since we are here I would also suggest adding multiple `targets` for these tests, it's good to have multiple `targets` since in the past involving multiple resources exposed various bugs. Right now, for `targets` we have only one element.

Reply:
added another target and also tested it's correct removal at the end